menas/src/main/scala/za/co/absa/enceladus/menas/repositories/RunMongoRepository.scala

```diff
@@ -281,6 +281,16 @@ class RunMongoRepository @Autowired()(mongoDb: MongoDatabase)
       .toFuture()
   }
 
+  def getRunBySparkAppId(appId: String): Future[Seq[Run]] = {
```

Comment:
Could you write some tests for this in the `RunRepositoryIntegrationSuite`?

Reply:
Added in 013c508 , thnx